hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/autoReloadPanel/autoReloadPanel.tsx

```diff
@@ -38,21 +42,47 @@ class AutoReloadPanel extends React.Component<IAutoReloadPanelProps> {
   };
 
   render() {
-    const {onReload, lastUpdated, isLoading} = this.props;
-    const lastUpdatedText = lastUpdated === 0 ? 'NA' :
+    const {onReload, lastUpdated, lastUpdatedOMDBDelta,lastUpdatedOMDBFull,isLoading,lastUpdatedOMDBDeltaText,lastUpdatedOMDBFullText} = this.props;
+    
+     const lastUpdatedText = lastUpdated === 0 || lastUpdated === undefined ? 'NA' :
       (
         <Tooltip
           placement='bottom' title={moment(lastUpdated).format('ll LTS')}
         >
-          {moment(lastUpdated).format('LTS')}
+          {moment(lastUpdated).format('LT')}
         </Tooltip>
       );
+
+      const omDBDeltaFullToolTip = <span>
+          {lastUpdatedOMDBFullText}: {moment(lastUpdatedOMDBFull).fromNow()}, {moment(lastUpdatedOMDBFull).format('LT')}
+          <br/>
+          {lastUpdatedOMDBDeltaText}: {moment(lastUpdatedOMDBDelta).fromNow()}, {moment(lastUpdatedOMDBDelta).format('LT')}
+      </span>
+
+      const lastUpdatedDeltaToolTip = lastUpdatedOMDBDelta === 0 || lastUpdatedOMDBDelta === undefined || lastUpdatedOMDBFull === 0 || lastUpdatedOMDBFull === undefined ? 'NA' :
```

Comment:
This is not necessarily the last updated delta, so we can use the previously used name like this (just so the code can be more understandable later).
```suggestion
      const lastUpdatedDeltaFullToolTip = lastUpdatedOMDBDelta === 0 || lastUpdatedOMDBDelta === undefined || lastUpdatedOMDBFull === 0 || lastUpdatedOMDBFull === undefined ? 'NA' :
```

Reply:
Done Changes as Suggested by You.